index.js

```diff
@@ -99,12 +100,29 @@ For example:
       `❌ Found an empty lcov file at "${path}".
 An empty lcov file was found but with no coverage data. This might be because \
 you have no test files or your tests are not generating any coverage data.
-`
+`,
     );
     return false;
   }
 
   return true;
 }
 
+function parseMinCoverage(input) {
```

Comment:
**Nit:** Rename the method to `tryParseMinCoverage` since by convention `parseFoo` methods throw an error and do not return null, whereas `tryParseFoo` do not throw an error and return null upon failure. The failure could be logged within the main code flow.

```suggestion
function parseMinCoverage(input) {
```